packages/client/src/util/CanaryExecutionStatusResponseFactory.ts

```diff
@@ -0,0 +1,112 @@
+import { CanaryExecutionStatusResponse } from '../domain/CanaryExecutionStatusResponse';
+import uuid from 'uuid/v1';
+
+export default class CanaryExecutionStatusResponseFactory {
```

Comment:
I don't think you want this file at all, you should be getting the response from the api, and shouldn't have to craft a stub manually.

Reply:
Refactored and removed this file